requirements.txt

```diff
@@ -1,5 +1,5 @@
 wheel
-aiohttp
+multidict
```

Comment:
Please add a specific version, to have a reproductive environment for developers.
multidict has already broken the compatibility between 1.0 and 2.0